### PR TITLE
Add hybrid app support

### PR DESCRIPTION
**Pull request by MrRefactor**
### Details

Hybrid app support as a follow-up from `react-native 0.75` upgrade - more context [here](https://github.com/Expensify/App/pull/48626#pullrequestreview-2282496318)

### Related Issues

GH_LINK

### CODE REVIEW

**Comment by MrRefactor** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by MrRefactor** [comment]
recheck

File: RNLiveMarkdown.podspec [hunk 1]
```diff
@@ -31,10 +31,10 @@ Pod::Spec.new do |s|
   install_modules_dependencies(s)
 
   if ENV['USE_FRAMEWORKS'] && ENV['RCT_NEW_ARCH_ENABLED']
```
Comment (tomekzaw) [inline RNLiveMarkdown.podspec:33]:
```suggestion
  if ENV['USE_FRAMEWORKS'] == 'static' && ENV['RCT_NEW_ARCH_ENABLED'] == '1'
```

File: RNLiveMarkdown.podspec [hunk 2]
```diff
@@ -31,10 +31,10 @@ Pod::Spec.new do |s|
...
-    add_dependency(s, "React-Fabric", :additional_framework_paths => [
+    add_dependency(s, "React-FabricComponents", :additional_framework_paths => [
       "react/renderer/textlayoutmanager/platform/ios",
-      "react/renderer/components/textinput/iostextinput",
-    ])
+      "react/renderer/components/textinput/platform/ios",
+    ]);
```
Comment (tomekzaw) [inline RNLiveMarkdown.podspec:37]:
```suggestion
    ])
```

**Review by tomekzaw — CHANGES_REQUESTED** [review]

**Review by tomekzaw — COMMENTED** [review]
Waiting for @war-in's confirmation that this PR works in HybridApp

File: RNLiveMarkdown.podspec [hunk 3]
```diff
@@ -30,10 +30,10 @@ Pod::Spec.new do |s|
 
   install_modules_dependencies(s)
 
-  if ENV['USE_FRAMEWORKS'] && ENV['RCT_NEW_ARCH_ENABLED']
-    add_dependency(s, "React-Fabric", :additional_framework_paths => [
+  if ENV['USE_FRAMEWORKS'] == 'static' && ENV['RCT_NEW_ARCH_ENABLED'] == '1'
```
Comment (tomekzaw) [inline RNLiveMarkdown.podspec:33]:
```suggestion
  if ENV['USE_FRAMEWORKS'] != nil && ENV['RCT_NEW_ARCH_ENABLED'] == '1'
```

Comment (tomekzaw) [inline RNLiveMarkdown.podspec:33]:
```suggestion
  if ENV['USE_FRAMEWORKS'] != nil && ENV['RCT_NEW_ARCH_ENABLED'] == '1'
```

Comment (tomekzaw) [inline RNLiveMarkdown.podspec:33]:
```suggestion
  if ENV['USE_FRAMEWORKS'] != nil && ENV['RCT_NEW_ARCH_ENABLED'] == '1'
```

**Review by tomekzaw — CHANGES_REQUESTED** [review]

**Review by tomekzaw — APPROVED** [review]

**Comment by war-in** [comment]
Confirming that PR works in HybridApp 👍

**Pull request merged**